contentcuration/contentcuration/tests/test_exportchannel.py

```diff
@@ -192,6 +193,34 @@ def test_create_slideshow_manifest(self):
         manifest_collection = cc.File.objects.filter(contentnode=ccnode, preset_id=u"slideshow_manifest")
         assert len(manifest_collection) == 1
 
+    def test_blocking_task_detection(self):
```

Comment:
Need to actually generate the user that you are using here to assign `user_id`, this is causing the test failures.